src/components/WindRose/WindRoseForm.tsx

```diff
@@ -0,0 +1,102 @@
+import { format } from 'date-fns';
+import { type Dispatch, type SetStateAction, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { type ChartData } from 'react-windrose-chart';
+
+import { chartData } from '@/components/WindRose/data';
+
+interface WindRoseFormProps {
+  setChartData: Dispatch<SetStateAction<ChartData[] | null>>;
+}
+
+type WindRoseFormData = {
+  dateFrom: string;
+  dateTo: string;
+  pollution: string;
+};
+
+const DATE_FORMAT = 'yyyy-MM-dd';
+
+const pollutionTypes = [
+  { value: 'PM25', label: 'PM2.5' },
+  { value: 'PM10', label: 'PM10' },
+  { value: 'CO', label: 'CO' },
+  { value: 'NO2', label: 'NO2' },
+  { value: 'O3', label: 'O3' },
+  { value: 'SO2', label: 'SO2' }
+];
```

Comment:
This copied from `TimeSeriesForm.tsx`. If we have the same values, we should probably move it to another file, so we don't duplicate it.